quickwit/quickwit-index-management/src/garbage_collection.rs

```diff
@@ -182,12 +182,12 @@ async fn delete_splits_marked_for_deletion(
     let mut removed_splits = Vec::new();
     let mut failed_splits = Vec::new();
 
-    'outer: loop {
-        let mut exit = false;
+    loop {
         let query = ListSplitsQuery::try_from_index_uids(index_uids.clone())?
```

Comment:
todo this is the only Err code path, and it shouldn't ever happen, we should just return an empty success as there was nothing to do